src/components/pages/maintenance.yml

```diff
@@ -11,6 +11,6 @@ maintenance:
   tagsTabTitle:
     en: Tags
     de: Tags
-  statisticsTabTitle:
-    en: Statistics
-    de: Statistiken
+  requestsTabTitle:
+    en: Requests
+    de: Zugriffe
```

Comment:
I found "Zugriffe" sounded a lot better than the "official" translation "Anforderungen".

Reply:
You are the German authority here, so it's your call 😁 
I can only say shorter words look better to me.